charts/kyuubi/values.yaml

```diff
@@ -116,13 +116,14 @@ server:
       nodePort: ~
       annotations: {}
 
+monitoring:
   # Exposes metrics in Prometheus format
   prometheus:
     enabled: true
     port: 10019
     service:
```

Comment:
Could you please check if the node `service` is not used anymore and deleted it?